app/views/BoxNumberView.scala.html

```diff
@@ -14,37 +14,39 @@
  * limitations under the License.
  *@
 
-@this(layout: templates.Layout,
+@this(
+        layout: templates.Layout,
         formHelper: FormWithCSRF,
-        h1: components.h1,
         errorSummary: components.errorSummary,
         button: components.button,
-        inputNumber: components.inputNumber
+        inputRadio: components.inputRadio
 )
 
-@(form: Form[_], backLink: Call)(implicit request: Request[_], messages: Messages)
-
-@layout(
-    pageTitle = messages("boxNumber.pageTitle"),
-    form = Some(form),
-    customBackLinkUrl = Some(backLink)) {
-
-    @errorSummary(form.errors)
-
-    @formHelper(action = controllers.routes.BoxNumberController.onSubmit()) {
-        @inputNumber(
-            form = form,
-            id = "value",
-            name = "value",
-            label = messages("boxNumber.heading"),
-            classes = Some("govuk-fieldset__legend govuk-input--width-5"),
-            isPageHeading = true
-        )
-        @button("common.button.continue")
-    }
+@(form: Form[_], backLink: Call, radioItems: Seq[RadioItem])(implicit request: Request[_], messages: Messages)
```

Comment:
Though `radioItems` is technically a sequence of radio items, it would be better to name the parameter after what it represents, in this case, `boxNumbers`.